src/registrar/management/commands/copy_names_from_contacts_to_users.py

```diff
@@ -0,0 +1,236 @@
+            {TerminalColors.ENDC}
+            """,
+        )
+
+    def print_summary_of_findings(
+        self,
+        skipped_contacts,
+        eligible_users,
+        processed_users,
+        debug_on,
+    ):
+        """Prints to terminal a summary of findings from
+        copying first and last names from contacts to users"""
+
+        total_eligible_users = len(eligible_users)
+        total_skipped_contacts = len(skipped_contacts)
+        total_processed_users = len(processed_users)
+
+        logger.info(
+            f"""{TerminalColors.OKGREEN}
+            ============= FINISHED ===============
+            Skipped {total_skipped_contacts} contacts
+            Found {total_eligible_users} users linked to contacts
+            Processed {total_processed_users} users
+            {TerminalColors.ENDC}
+            """  # noqa
+        )
+
+        # DEBUG:
+        TerminalHelper.print_conditional(
+            debug_on,
+            f"""{TerminalColors.YELLOW}
+            ======= DEBUG OUTPUT =======
+            Users who have a linked contact:
+            {eligible_users}
+
+            Processed users (users who have a linked contact and a missing first or last name):
+            {processed_users}
+
+            ===== SKIPPED CONTACTS =====
+            {skipped_contacts}
+
+            {TerminalColors.ENDC}
+            """,
+        )
+
+    # ======================================================
+    # ===================    USER    =====================
+    # ======================================================
+    def update_user(self, contact: Contact, debug_on: bool):
+        """Given a contact with a first_name and last_name, find & update an existing
+        corresponding user if her first_name and last_name are null.
+
+        Returns the corresponding User object.
+        """
+
+        user_exists = User.objects.filter(contact=contact).exists()
+        if user_exists:
+            try:
+                # ----------------------- UPDATE USER -----------------------
+                # ---- GET THE USER
+                eligible_user = User.objects.get(contact=contact)
+                processed_user = None
+                # DEBUG:
+                TerminalHelper.print_conditional(
+                    debug_on,
+                    f"""{TerminalColors.YELLOW}
+                    > Found linked user for contact:
+                    {contact} {contact.email} {contact.first_name} {contact.last_name}
+                    > The linked user is {eligible_user} {eligible_user.username}
+                    {TerminalColors.ENDC}""",  # noqa
+                )
+
+                # ---- UPDATE THE USER IF IT DOES NOT HAVE A FIRST AND LAST NAMES
+                # ---- LET'S KEEP A LIGHT TOUCH
+                if not eligible_user.first_name and not eligible_user.last_name:
+                    # (expression has type "str | None", variable has type "str | int | Combinable")
```

Comment:
That is odd. I've ran into similar issues in the past with the linter. I think ignoring this is fine as long as there is no foreseeable way that this can cause issues, and I think this is probably the case given the db definition.